### ISSUE TITLE
Update SB Aliases

### ISSUE THREAD

**Issue by Barrow1990** [saltyorg/docs#258]
Update SB Aliases description under: https://docs.saltbox.dev/reference/guides/aliases/?h=

Current available options:

- sb update
- sb list 
- sb install <tag>
- sb bench
- sb recreate-venv
- sb inventory
- sb branch <branch>
- sb sandbox-branch <branch>

**Comment by owine** [comment]
@saltydk do you want all of these documented?

**Comment by keldian** [comment]
I'm thinking we could use a page dedicated to `sb`, tags and roles since right now that info is scattered all over the docs. For instance, the new custom meta-tag stuff could be offloaded there rather than be footnotes on https://docs.saltbox.dev/saltbox/basics/install_types/. The inventory page might have less things to cover as a result and be more concise.

I can work on it if everyone's in agreement.

**Comment by owine** [comment]
Makes sense to me and timing is good with the new binary rewrite deployed

**Comment by Barrow1990** [comment]
Hi, Is there an update on this?
Also do we know what sb sandbox-branch does and is this the best method for doing the following.
[Contributing to Sandbox Apps](https://docs.saltbox.dev/sandbox/basics/#contributing-to-sandbox-apps)

**Comment by saltydk** [comment]
branch commands are only useful for when users test something we don't think is ready for the main branch.

**Comment by saltydk** [comment]
Frankly, as far as I am concerned the sb tool describes what all commands do, if users need more they can ask on discord.

**Comment by Barrow1990** [comment]
That's fine. Should this issue be removed then?

**Comment by saltydk** [comment]
Docs could just reference running
```
sb --help
```
since that will be the most up to date descriptions of all of the commands, if something is too vague then issue belongs in the sb repo.
